### PR TITLE
Feat/implement socket io from left panel and added interfaces

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
Feat/implement socket io from left panel and added interfaces

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: apps/@sparrow-desktop/src/models/collection.model.ts [hunk 1]
```diff
@@ -86,6 +86,27 @@ const websocketItems = {
...
 };
+
+const socketioItems = {
+  url: {
+    type: "string",
+  },
+  message: {
+    type: "string",
+  },
+  selectedWebSocketBodyType: {
```
Comment (Astitva877) [inline apps/@sparrow-desktop/src/models/collection.model.ts:97]: we will also have events here.

File: apps/@sparrow-desktop/src/models/tab.model.ts [hunk 2]
```diff
@@ -361,7 +356,95 @@ export const tabSchemaLiteral = {
...
+                  type: "string",
+                },
+                isHeaderBulkEditActive: {
+                  type: "string",
+                },
+              },
+            },
+          },
+        },
+        socketio: {
```
Comment (Astitva877) [inline apps/@sparrow-desktop/src/models/tab.model.ts:447]: here also we don't have events in socketio

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: apps/@sparrow-desktop/src/models/tab.model.ts (hunk 2 above)
Comment (itsmdasifraza) [reply apps/@sparrow-desktop/src/models/tab.model.ts:447]: resolved

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**